exampleSite/static/admin/config.yml

```diff
@@ -48,6 +48,24 @@ collections:
         widget: "object"
         fields:
           - {label: "Discord Name", name: "discord_id", widget: "string"}
+  # encounter metadata
+  - label: "Encounter metadata"
+    name: "encounter-guide"
+    folder: "data/encounter"
+    format: "json"
+    identifier_field: "category"
+    create: true
+    fields: {
+      - {label: "Category", name: "category", widget: "string"}
+      - label: "Encounters" 
+        name: "encounters"
+        widget: "list"
+        fields:
+          - {label: "Identifier", name: "id", widget: "string"}
+          - {label: "Name of Encounter", name: "name", widget: "string"}
+          - {label: "Image Link", name: "image_link", widget: "string"}
+          - {label: "Encounter Link", name: "encounter_link", widget: "string"}
+    }
```

Comment:
I'm seeing this when I use hugo to render and then go to http://localhost:1313/admin/#/:

<img width="751" alt="Screen Shot 2021-11-05 at 6 17 37 PM" src="https://user-images.githubusercontent.com/83298827/140588395-8c2adc4a-0ddf-4b69-a382-12b7d98ec619.png">

I believe it's because you're doing json-like `{}` after the `fields:` view; maybe something more like this?

```yaml
fields:
  - {label: "Identifier", name: "id", widget: "string"}
  - {label: "Name of Encounter", name: "name", widget: "string"}
  - {label: "Image Link", name: "image_link", widget: "string"}
  - {label: "Encounter Link", name: "encounter_link", widget: "string"}
  ```
I might also switch the string widget out for `select` widget (docs [here](https://www.netlifycms.org/docs/widgets/#select)) – this way, we can narrow the category of file(s).

tl;dr - yaml is weird and we should probably make a PR checker that'll yamllint this file just like we have on the production repo.

Reply:
ahhh my bad. You're correct; the `fields:` shouldn't start with `{` oops! I'll try to test this on the admin page before I push the change.

Reply:
@nonowazu fixed in commit https://github.com/The-Balance-FFXIV/glam/pull/63/commits/f9fbce0568dc5a840fde1d8445d24742fc61b1d3